config/locales/en.yml

```
@@ -1320,23 +1320,24 @@ instructions.forgot_password.close_window: You can close this browser window onc
 instructions.mfa.piv_cac.add_from_sign_in_html: '<strong>Instructions:</strong> Insert your PIV or CAC on <strong>“ADD PIV/CAC”</strong>. You’ll need to <strong>choose a certificate</strong> (the right one likely has your name of it) and <strong>enter your PIN</strong> (your PIN was created when you set up your PIV/CAC).'
 instructions.mfa.piv_cac.already_associated_html: Please choose a certificate from a different PIV/CAC, contact your administrator to ensure your PIV/CAC is up to date. If you think this is an error, %{try_again_html}.
 instructions.mfa.piv_cac.back_to_sign_in: Go back to sign in
-instructions.mfa.piv_cac.confirm_piv_cac: Present the PIV/CAC that you associated with your account.
-instructions.mfa.piv_cac.did_not_work_html: Please %{please_try_again_html}. If this problem continues, contact your agency administrator.
+instructions.mfa.piv_cac.confirm_piv_cac: Insert your PIV/CAC that you associated with your account into a smart card reader.
+instructions.mfa.piv_cac.did_not_work_html: Please make sure your PIV/CAC is properly inserted into your smart card reader and %{please_try_again_html}. If this problem continues, contact your agency administrator.
 instructions.mfa.piv_cac.http_failure: The server took too long to respond. Please try again.
-instructions.mfa.piv_cac.no_certificate_html: Please make sure your PIV/CAC is connected and %{try_again_html}. If this problem continues, contact your agency administrator.
+instructions.mfa.piv_cac.no_certificate_html: Please make sure your PIV/CAC is properly inserted into your smart card reader and %{try_again_html}. If this problem continues, contact your agency administrator.
 instructions.mfa.piv_cac.not_auth_cert_html: The certificate you selected is invalid for this account. Please %{please_try_again_html} with a different certificate. If this problem continues, contact your agency administrator.
 instructions.mfa.piv_cac.please_try_again: try again
-instructions.mfa.piv_cac.sign_in_html: Make sure <strong>you have a %{app_name} account</strong> and <strong>you’ve set up PIV/CAC</strong> as a two-factor authentication method.
+instructions.mfa.piv_cac.sign_in: Make sure you have a %{app_name} account and you’ve set up your Personal Identity Verification (PIV) or Common Access Card (CAC) as a two-factor authentication method.
 instructions.mfa.piv_cac.step_1: Give it a nickname
 instructions.mfa.piv_cac.step_1_info: If you add more than one PIV/CAC, you’ll know which one’s which.
 instructions.mfa.piv_cac.step_2: Insert your PIV/CAC into your card reader
 instructions.mfa.piv_cac.step_3: Add your PIV/CAC
 instructions.mfa.piv_cac.step_3_info_html: You’ll need to <strong>choose a certificate</strong> (the right one likely has your name in it) and <strong>enter your PIN</strong> (your PIN was created when you set up your PIV/CAC).
 instructions.mfa.piv_cac.try_again: try again
+instructions.mfa.piv_cac.try_again_capitalized: Try again
```

Our string key naming conventions aren't super consistent, but I think the "instructions" top-level namespace here is usually reserved for paragraph text, and we have e.g. `forms.piv_cac_mfa.submit` for button labels. Naming it like those would also avoid having to disambiguate between the capitalized and uncapitalized string fragments.

```suggestion
forms.piv_cac_setup.try_again: Try again
```

I also think "Try again" is such a common button label that we should probably just have something super-generic for it similar to `links.cancel`, but that might be a bigger effort than we'd want to take on here.